packages/mapviewer/src/modules/drawing/DrawingModule.vue

```diff
@@ -99,6 +102,11 @@ const isDrawingModified = computed(() => {
 watch(projection, () => {
     drawingLayer.setSource(createSourceForProjection())
 })
+watch(hasLoaded, () => {
+    if (hasLoaded.value) {
+        addKmlToDrawing()
+    }
+})
```

Comment:
would it be a good idea to set a `{once: true}` parameter to the watcher? hasLoaded is either always true (we had already loaded the layer before entering the drawing module), or false, then true (we enter the drawing module with the admin link or we created a new kml layer when starting to draw.)

If you don't think it's useful, we can avoid that :)

Reply:
Don't think that would greatly improve perf, and I'd really like to fix this bug ASAP. Let's go with the current state